doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1324 @@
+    * ``SB_CONFIG_SECURE_APPROTECT_USER_HANDLING`` for the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING` Kconfig option.
+    * ``SB_CONFIG_SECURE_APPROTECT_USE_UICR`` for the :kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USE_UICR` Kconfig option.
+
+  * CMake warning when static partition manager file has been changed but changes will not be read until a pristine build is performed.
+  * Encrypted firmware update to :file:`dfu_application.zip` instead of the unencrypted firmware update when encrypted image support is enabled in sysbuild.
+  * Sysbuild-assigned MCUboot image IDs feature, which deals with MCUboot image IDs for different components in a project.
+  * The ``SB_CONFIG_LWM2M_CARRIER_DIVIDED_DFU`` sysbuild Kconfig option that enables the generation of proprietary application update files required for the LwM2M carrier divided FOTA procedure.
+
+* Fixed:
+
+  * An issue with the |NCS| boot banner using ``CMAKE_SOURCE_DIR`` instead of ``APPLICATION_SOURCE_DIR``.
+  * An issue with :kconfig:option:`CONFIG_MBEDTLS_CFG_FILE` having a prompt which would, in some circumstances, lead to a sticky value if the configuration was changed.
+  * An issue with changes to image configurations in a ``sysbuild.cmake`` file not being applied to variant images.
+  * An issue with signing variables not expanding strings for nRF70 Series devices.
+
+* Deprecated the following devicetree properties:
+
+  * ``owner-id``
+  * ``perm-read``
+  * ``perm-write``
+  * ``perm-execute``
+  * ``perm-secure``
+  * ``non-secure-callable``
+
+  It is recommended to replace them with the new devicetree property: ``nordic,access``.
+  See the :ref:`migration guide <migration_2.8_recommended>` for more information.
+
+* Removed the non-working support for configuring the NSIB signing key through the environmental or command line variable (``SB_SIGNING_KEY_FILE``) along with child image.
+
+  .. note::
+     This feature has never been functional.
+     To configure the signing key, use any available Kconfig method.
+
+Bootloaders and DFU
+===================
+
+* Added:
+
+  * ``SB_CONFIG_MCUBOOT_NRF53_MULTI_IMAGE_UPDATE`` sysbuild Kconfig to allow setting network core application updates for MCUboot.
+  * Direct-XIP (without revert) support to :ref:`QSPI split image <qspi_xip_split_image>` for sysbuild.
+  * Documentation for :ref:`mcuboot_image_compression`.
+  * Documentation for :ref:`qspi_xip_split_image` functionality.
+  * Documenetation for :ref:`sysbuild_assigned_images_ids`.
+  * A section in the sysbuild-related migration guide about the migration of :ref:`child_parent_to_sysbuild_migration_qspi_xip` from child/parent image to sysbuild.
+
+* Updated the procedure for signing the application image built for booting by MCUboot in direct-XIP mode with revert support.
+  Now, the Intel-Hex file of the application image automatically receives a confirmation flag.
```

Comment:
```suggestion
* Updated: 

  * The procedure for signing the application image built for booting by MCUboot in direct-XIP mode with revert support.
    Now, the Intel-HEX file of the application image automatically receives a confirmation flag.
  * The allowed offset for :ref:`doc_fw_info` in the :ref:`bootloader`.
    It can now be placed at offset 0x600, however, it cannot be used for any applications with |NSIB| compiled before this change.
```

Reply:
@annwoj FYI - I have move the changelog entry of PR https://github.com/nrfconnect/sdk-nrf/pull/16765 from the release notes to Change log to main -  https://github.com/nrfconnect/sdk-nrf/compare/780a0758b3774b1bfc963e86c19c0954d6c2252f..41f93f0b887ebeb166cbf4c7d24cd398642d841c, since the PR is not part of the 2.8 release now.